### PR TITLE
adding qts and rewording copy

### PR DESCRIPTION
**Pull request by EllieNodder**
### Trello card

https://trello.com/c/5HOaD98u/3443-update-steps-page-to-reference-qts

### Context

Steps currently doesn't reference QTS, which isn't good for challenging the misconception users have that it's a PGCE they need to teach.

Some of the copy also currently isn't accurate.

### CODE REVIEW

**Comment by EllieNodder** [comment]
@gemmadallmandfe @GDW7 have edited some of the copy so that:

- the content is more accurate and clear
- the links are accessible
- we now mention qualified teacher status (QTS)

In rewriting some of the content under step 4, I've also got rid of 'PGCE'. I think we _could_ still talk about PGCEs if we wanted, but it's not really necessary for this page. And from an SEO/keywords point of view, if people are searching 'PGCE', we want them to land on the PGCE/train to be a teacher/maybe QTS page, rather than steps, so don't see the value in stuffing it in.

But happy for you to disagree and can always add it back in!

**Comment by GDW7** [comment]
Hi @EllieNodder that sounds sensible and the changes look good to me.

**Review by gemmadallmandfe — COMMENTED** [review]
Thanks @EllieNodder - just left one minor comment. Happy for this to be deployed to prod when you've had a look at this (feel free to disagree!).

Re taking out PGCE - I think that sounds reasonable. Looking in GSC, search queries with 'pgce' don't tend to end up on Steps. I've also had a look at ad campaign data in GA and their PGCE campaigns don't point to Steps either.

I think we just need to inform the advertising team when this goes live.

File: app/views/content/steps-to-become-a-teacher/_step_2_check_your_qualifications.html.erb [hunk 1]
```diff
@@ -1,11 +1,13 @@
-<p>Once you have a degree or equivalent qualification you’re ready for postgraduate primary or secondary initial teacher training courses.</p>
+<p>If you have a degree or equivalent qualification, you can do postgraduate primary or secondary teacher training.</p>
```
[the remark this answers is not in the document]
Comment (EllieNodder) [reply app/views/content/steps-to-become-a-teacher/_step_2_check_your_qualifications.html.erb:1]: Does that suggest then that you can't apply until you've finished your degree though? (Or am I overthinking!)

[the remark this answers is not in the document]
Comment (gemmadallmandfe) [reply app/views/content/steps-to-become-a-teacher/_step_2_check_your_qualifications.html.erb:1]: Yes I think you are right - let's leave as is :-)

File: app/views/content/steps-to-become-a-teacher/_step_2_check_your_qualifications.html.erb [hunk 2]
```diff
@@ -1,11 +1,13 @@
...
 
-<p>You also need the following GCSEs at grade 4 (C) or above, or equivalent qualifications:</p>
+<p>If you do not have a degree and are not studying for one, you can do <a href="/train-to-be-a-teacher/if-you-dont-have-a-degree">undergraduate teacher training</a>.</p>
```
Comment (gemmadallmandfe) [inline app/views/content/steps-to-become-a-teacher/_step_2_check_your_qualifications.html.erb:3]: @EllieNodder Is this undergrad sentence in the right place? It is now in the middle of the postgrad info - is that confusing?

Comment (EllieNodder) [reply app/views/content/steps-to-become-a-teacher/_step_2_check_your_qualifications.html.erb:3]: My thinking was that the qualifications info was relevant for both postgrad and undergrad courses. And as a user, if I see 'if you have a degree', I'd then be thinking 'what about if I don't have a degree?' so thought it made sense to move that sentence further up. But equally happy to move it back down!

Comment (gemmadallmandfe) [reply app/views/content/steps-to-become-a-teacher/_step_2_check_your_qualifications.html.erb:3]: If the qualifications section also applies to undergrad degrees then let's leave it as is

**Review by gemmadallmandfe — APPROVED** [review]
Looks good to me

**Pull request merged**